lxd/storage/backend_lxd.go

```diff
@@ -2555,6 +2540,81 @@ func (b *lxdBackend) CreateInstanceFromConversion(inst instance.Instance, conn i
 
 	revert.Add(func() { _ = b.driver.DeleteVolume(volCopy.Volume, op) })
 
+	// At this point, the instance's volume is populated. If "virtio" option is enabled,
+	// inject the virtio drivers.
+	if slices.Contains(args.ConversionOptions, "virtio") {
+		b.logger.Debug("Inject virtio drivers started")
+		defer b.logger.Debug("Inject virtio drivers finished")
+
+		err = b.driver.MountVolume(vol, op)
+		if err != nil {
+			return err
+		}
+
+		defer func() { _, _ = b.driver.UnmountVolume(vol, true, op) }()
+
+		diskPath, err := b.driver.GetVolumeDiskPath(vol)
+		if err != nil {
+			return err
+		}
+
+		out, err := exec.Command("virt-v2v-in-place", "--version").CombinedOutput()
+		if err != nil {
+			return fmt.Errorf("Failed to get virt-v2v-in-place version (%v): %s", err, string(out))
+		}
+
+		// checkMinVersions extracts version numbers and compares them one by one returning
+		// true if given version is larger then or equal to minimum version and false otherwise.
+		checkMinVersion := func(version string, min string) bool {
```

Comment:
Can you use `version.NewDottedVersion()` for this comparison?

Reply:
I knew there must already be something for comparing versions. Thanks :)